cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -3341,35 +3337,35 @@
 
 ;; Helper for creating `packsswb` instructions.
 (decl x64_packsswb (Xmm XmmMem) Xmm)
-(rule 0 (x64_packsswb src1 src2)
-      (xmm_rm_r (SseOpcode.Packsswb) src1 src2))
 (rule 1 (x64_packsswb src1 src2)
         (if-let true (use_avx))
         (xmm_rmir_vex (AvxOpcode.Vpacksswb) src1 src2))
+(rule 0 (x64_packsswb src1 src2)
+      (x64_packsswb_a src1 src2))
```

Comment:
Now that the alternate stuff is built, we can just use one rule that uses the "or_avx" ISLE rule.